poetry.lock

```diff
@@ -1,4 +1,4 @@
-# This file is automatically @generated by Poetry 1.4.1 and should not be changed by hand.
```

Comment:
Probably not a problem, but I don't want to lower the version.

Reply:
re-locked with the latest poetry 1.4.2 in [47d27fd](https://github.com/m3dev/kannon/pull/27/commits/47d27fdee775c5e9ea08cef98a42b1305f09f4d9)